Src/coffeescript/cql-execution/src/example/browser/cql4browsers.js

```diff
@@ -3626,7 +3626,7 @@
       ref = this.element;
       for (i = 0, len = ref.length; i < len; i++) {
         el = ref[i];
-        obj[el.name] = el.execute(ctx);
+        obj[el.name] = el.exec(ctx);
```

Comment:
This is on a previous PR that was merged. Want me to add it to the correct coffee script file and add the comment to this PR?